scripts/kind.sh

```diff
@@ -42,7 +68,7 @@ _create_kind_cluster() {
             kind create cluster --name "$__cluster_name" \
                 ${cluster_version:+ --image kindest/node:v$cluster_version} \
                 --config "$config_file_path" \
-                --kubeconfig $__kubeconfig_path 1>&2 || :
+                --kubeconfig $__kubeconfig_path 1>&2 2>/dev/null || :
```

Comment:
I chose to not pipe stderr to dev null because if the KIND cluster fails, I wanted the error to be shown to the user. But I know it dumps a lot of useless warnings into stderr so I also understand why we'd want to suppress those. Thoughts?

Reply:
Yeah... I get really peeved when tools dump non-error output to `stderr` :( Especially output filled with emoticons...

I can revert this change if you really want.

Reply:
😭 Yeah if we could just keep it, we've had so many problems with KIND in the past, I'm jaded 😂 🙌 🙏

Reply:
reverted.

Reply:
> sob Yeah if we could just keep it, we've had so many problems with KIND in the past, I'm jaded joy raised_hands pray

I see what you did there. Very funny. :)